roofit/roofitcore/src/RooEfficiency.cxx

```diff
@@ -111,6 +111,11 @@ double RooEfficiency::analyticalIntegral(Int_t code, const char* /*rangeName*/)
   return 1.0 ;
 }
 
+void RooEfficiency::translate(RooFit::Detail::CodeSquashContext &ctx) const
+{
+    int sigCatIndex = _cat.arg().lookupIndex(_sigCatName.Data());
+    ctx.addResult(this, ctx.buildCall("RooFit::Detail::EvaluateFuncs::RooEfficiencyEvaluate",  _effFunc, _cat, sigCatIndex));
```

Comment:
```suggestion
    ctx.addResult(this, ctx.buildCall("RooFit::Detail::EvaluateFuncs::efficiencyEvaluate",  _effFunc, _cat, sigCatIndex));
```

@guitargeek, I realize that `RooEfficiency::evaluate` is quite different. What can we do to reduce the code duplication here?

Reply:
It was mathematically not different, and I updated the PR to use the new `efficiencyEvaluate` also in `RooEfficiency::evaluate`.